src/ZPublisher/HTTPResponse.py

```diff
@@ -903,6 +903,8 @@ def _cookie_list(self):
                 # and block read/write access via JavaScript
                 elif name == 'http_only' and v:
                     cookie = '%s; HTTPOnly' % cookie
+                elif name == 'same_site':
+                    cookie = '%s; SameSite=%s' % (cookie, v)
```

Comment:
Please add a comment above line 906 explaining the purpose, and linking to the RFC.